prelude/__internal__/Function.mad

```diff
@@ -134,6 +134,18 @@ export all = (predicate, list) => where(list) {
       : all(predicate, xs)
 }
 
+/**
+ * Functional "or"
+ */
+either :: (a -> Boolean) -> (a -> Boolean) -> a -> Boolean
+export either = (predA, predB, x) => predA(x) || predB(x)
+
+/**
+ * Functional "and"
+ */
+both :: (a -> Boolean) -> (a -> Boolean) -> a -> Boolean
+export both = (predA, predB, x) => predA(x) && predB(x)
```

Comment:
Should these be called `or` and `and`?

Reply:
I considered that but a) these are the names `ramda` uses so I already have familiarity with these names and b) I'd kinda expect that `and` would be `(a, b) => a && b` rather than a predicated thing.

Reply:
That makes sense!